models/spleen_deepedit_annotation/docs/README.md

```diff
@@ -74,6 +74,12 @@ This result is benchmarked under:
  - CUDA version: 12.0
  - GPU models and configuration: A100 80G
 
+RAM consumption:
+
+This bundle uses CacheDataset by default, the given dataset is the MSD Task09 Spleen.
```

Comment:
Thanks for the updates @tangy5 . Here I think we may not need to mention the dataset as it is described in `## Data`. What do you think?

Reply:
If we do not mention the dataset, how do users know the number of cached images? I feel it's better to add it here since the RAM consumption is closely associated with the dataset. Though we have a separate section for describing the dataset.

Reply:
Hi @zephyrie ,

Could you please help review this README doc update?

Thanks in advance.

Reply:
Here is my suggestion:

### Memory Consumption
- Dataset Manager: CacheDataset
- Data Size: 61 3D Volumes
- Cache Rate: 1.0
- Single GPU VRAM: 8.2G
- Multi GPU VRAM: 65G (8 GPUs)

Reply:
might be `RAM` rather than `GPU VRAM`